libs/core/src/lib/breadcrumb/breadcrumb.component.ts

```diff
@@ -135,10 +135,8 @@ export class BreadcrumbComponent implements OnInit, AfterViewInit {
      * if original element had router link we are proxying click to that element
      * */
     itemClicked(breadcrumbItem: BreadcrumbItemComponent, $event: any): void {
```

Comment:
if conditional check is no longer needed, please update function doc descriptions too.